client/app.jsx

```diff
@@ -23,16 +23,16 @@ class App extends React.Component {
     var listingId = this.state.listingId
     axios.get(`/api/reservation/${listingId}`)
       .then(res => {
-        console.log(res.data)
+        console.log(res.data.weeklyDiscount)
 
         this.setState({
           listingId: res.data.listingId,
           availableDates: res.data.availableDates,
           standardPrice: res.data.standardPrice,
-          cleaningFfee: res.data.cleaningFee,
+          cleaningFee: res.data.cleaningFee,
           weeklyDiscount: res.data.weeklyDiscount,
-          refundable: res.data.refundable
-        })
+          refundable: !!res.data.refundable
```

Comment:
The use of the boolean column in mysql changes true and false values to binary 0's & 1's.  This is a quick fix that converts untrue values (like 0) to false & true values (like 1) to true.  

Now that you mention it, this might be something better done on the server side before sending back on the API response. 🤔

Reply:
Oh i see. And yeah I agree, thats a good thought, separating the concerns re: data manipulation